src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -88,7 +93,7 @@ protected function configure()
 message.
 
 Example running against a Bundle (AcmeBundle)
-  <info>php %command.full_name% --dump-messages en AcmeBundle</info>
+  <info>php %command.full_name% --dump-messages --sort=asc en AcmeBundle</info>
```

Comment:
I think you can add another example to explain the sort option.

Reply:
two are not enough?